recipe/meta.yaml

```
@@ -23,25 +23,17 @@ package:
 source:
   # Using github as source for mlflow-skinny which does not exist on PyPI
   url: https://github.com/mlflow/mlflow/archive/refs/tags/v{{ version }}.zip
-  sha256: 910ad1e2fce93b412063637d465e9291c99678a3bbccbf2b4f7736887321f44b
+  sha256: 9cfa071e7a75ff4a0d47a3ee4bfc948a6dfa019edc2863325b71d2362f61b760
 
 build:
   number: 0
-  skip: true  # [py<39 or s390x]
-  script: |
-{% if mlflow_variant == "skinny" %}
+  skip: true  # [py<310 or s390x]
+  script: |  # [not win]
+    {% if mlflow_variant == "skinny" %}  # [not win]
```

```suggestion
  script: |
    {% if mlflow_variant == "skinny" %}
```
You should be able to do something like `{% if not win and mlflow_variant == "skinny" %}` to avoid having to do all the `# [no win]` preprocessors; but, we also still want to build `mlflow-skinny` on Windows, right? Should just be able to do the export differently per platform type (`unix` vs `win`) and then let the rest of the build process remain the same.

---

Thanks for the suggestion! I agree this way could be cleaner. Though for the Win build, we still need to use bld.bat to handle the symlink materialization issue. I simplified the recipe to remove the repeated # [not win] comments.